.github/workflows/tests_lmps_tncuda_python.yml

```diff
@@ -129,8 +130,8 @@ jobs:
           echo "python=$py_path" >> $GITHUB_OUTPUT
 
           pip_path=$(which python)
```

Comment:
Why do we need to create this new variable exactly equal to `py_path`?
Can't we do everything in terms of `py_path` only?